.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "env": {
+    "cjs": {
+      "presets": ["es2015", "react", "stage-2"],
```

Comment:
please, yes. Adding in JSX means we suddenly need many more MBs in dependencies and precompile conversion for a single line of code =)